docs/content/develop/add-resource.md

```diff
@@ -21,9 +21,10 @@ aliases:
   - /how-to
   - /docs/getting-started/provider-documentation
   - /getting-started/provider-documentation
+  - /docs/develop/resource
```

Comment:
```suggestion
  - /develop/resource
```

A long time ago there was a `/docs` folder _inside_ `docs/content` but it was unnecessary so it got removed. The path for aliases starts from inside `content`